hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -601,6 +605,13 @@ def validate_health_check_suppressions(suppressions):
                 "Non-HealthCheck value %r of type %s is invalid in suppress_health_check."
                 % (s, type(s).__name__)
             )
+        if s is HealthCheck.hung_test:
+            note_deprecation(
+                "HealthCheck.hung_test is deprecated and has no "
+                "effect, as we no longer run this health check.  "
+                "Use the max_examples and deadline settings togther instead.",
```

Comment:
I don't think `Use the max_examples and deadline settings togther instead.` is useful here given that they're *suppressing* this health check - they probably don't want the behaviour at all!